src/components/ButtonPanel.js

```diff
@@ -0,0 +1,33 @@
+import Button from './Button';
+
+const ButtonPanel = () => {
+  const groups = {
+    group1: ['AC', '+/-', '%', '÷'],
+    group2: ['7', '8', '9', 'x'],
+    group3: ['4', '5', '6', '-'],
+    group4: ['1', '2', '3', '+'],
+    group5: ['0', '.', '='],
+  };
+
+  return (
+    <>
+      <div>
+        {groups.group1.map(value => <Button key={value} name={value} />)}
```

Comment:
- [ ] **[OPTIONAL]** Excellent job here to DRY your code. We can make it more dry by using [`object.keys`](https://attacomsian.com/blog/javascript-iterate-objects#objectkeys-method). The outer loop will be for groups and the inner loop will be for group items.

Reply:
Thanks for the tip. I have implemented it. It looks very nice now.